mungegithub/mungers/sync/issue-sync.go

```diff
@@ -50,7 +50,7 @@ type Priority int
 
 // String return the priority label in string
 func (p Priority) String() string {
-	return fmt.Sprintf(priorityPrefix+"%d", p)
+	return "priority/failing-test"
```

Comment:
I think I see what you are getting at. Doing something like this would eliminate calls to the unnecessary auto-prioritization code paths. I think we would still need to change the String method to handle the new label since the sync.Priority type is just a wrapper around int. 

I did not want to get too deep into removing the auto-prioritization code for this PR unless you requested it.  From what I can tell the Priority type in issue-sync.go and flake-manager.go is not used anywhere else. So we may be able to remove or refactor it out somehow now that there is only one real value for it.